packages/dds/sequence/src/revertibles.ts

```diff
@@ -127,18 +128,16 @@ export function appendDeleteIntervalToRevertibles(
 	if (!startSeg) {
 		return revertibles;
 	}
-	const startType =
-		startSeg.removedSeq !== undefined
-			? ReferenceType.SlideOnRemove | ReferenceType.RangeBegin
-			: ReferenceType.StayOnRemove | ReferenceType.RangeBegin;
+	const startType = segmentIsRemoved(startSeg)
+		? ReferenceType.SlideOnRemove | ReferenceType.RangeBegin
+		: ReferenceType.StayOnRemove | ReferenceType.RangeBegin;
 	const endSeg = interval.end.getSegment() as SharedStringSegment | undefined;
 	if (!endSeg) {
 		return revertibles;
 	}
-	const endType =
-		endSeg.removedSeq !== undefined
-			? ReferenceType.SlideOnRemove | ReferenceType.RangeEnd
-			: ReferenceType.StayOnRemove | ReferenceType.RangeEnd;
+	const endType = segmentIsRemoved(startSeg)
```

Comment:
```suggestion
	const endType = segmentIsRemoved(endSeg)
```
Looks like a copy paste error (below case as well)